enterprise/frontend/src/metabase-enterprise/transforms-python/pages/PythonLibraryEditorPage/PythonLibraryEditorPage.tsx

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useState } from "react";
+import { t } from "ttag";
+
+import Button from "metabase/common/components/Button";
```

Comment:
@romeovs @ranquild Any particular reason we had to use the deprecated `Button` here instead of the Mantine one?